### DISCUSSION TITLE
Success on OpenCore AMD FX-4350 + 8800 GS

### DISCUSSION THREAD

**Discussion by w32u** [chris1111/Legacy-Video-patch#38] [Show and tell]
After the second attempt my 8800 GS successfully started on 10.15.7 with OpenCore

I followed these steps to install the patch:

1. Install macOS Catalina
2. Install all updates
3. Patch NVIDIA GPU like it's said on https://dortania.github.io/OpenCore-Post-Install/gpu-patching/nvidia-patching/
4. Reboot and check if it's patched (model and VRAM size are correct in About this Mac window)
5. [Disable SIP](https://dortania.github.io/OpenCore-Install-Guide/troubleshooting/extended/post-issues.html#disabling-sip) and set SecureBootModel to Disabled as described [here](https://dortania.github.io/OpenCore-Install-Guide/troubleshooting/extended/kernel-issues.html#stuck-on-ocb-loadimage-failed-security-violation)
6. Reboot and check if SIP is disabled. In terminal type `csrutil status`, you should get something like that:
![Screenshot 2021-04-09 at 23 16 10](https://user-images.githubusercontent.com/40323117/114235952-9556e000-9989-11eb-9323-5c09d0333b28.png)
7. Mount system partition as r/w with this command: `sudo mount -uw /`
8. Install Legacy Video Patch as described in the two videos
9. Reboot and enjoy

# Some photos
![Screenshot 2021-04-09 at 23 23 21](https://user-images.githubusercontent.com/40323117/114236665-9f2d1300-998a-11eb-968d-a6a872b5a78f.png)
![Screenshot 2021-04-09 at 23 26 12](https://user-images.githubusercontent.com/40323117/114236920-fb903280-998a-11eb-837d-c8b755ad5ae4.png)
![compressed](https://user-images.githubusercontent.com/40323117/114237166-5aee4280-998b-11eb-910e-c94a62ed81f9.jpeg)
Two monitors work (there are two DVI-I port and S-Video port on the graphics card, the fist monitor is connected with VGA via DVI-I - VGA adapter, the second one - with DVI-D cable)

Hope this guide will help anyone using OpenCore and getting kernel panics. Cheers!

**Comment by chris1111** [comment]
Thanks for the details but sudo mount -uw / is not necessary because is already use by the program
See the [POST script ](https://github.com/chris1111/Legacy-Video-patch/blob/master/Sources/POST)
